numpy/_core/tests/test_umath.py

```diff
@@ -1258,6 +1258,17 @@ def test_power_fast_paths(self):
             result = np.power(a, 0.5)
             assert_array_max_ulp(result, expected, maxulp=1)
 
+    def test_large_fast_power(self):
+        # gh-28248
+        for dt in [np.float32, np.float64]:
+            a = np.random.uniform(1, 4, 1000).astype(dt)
```

Comment:
might be good to pin the random seed/Generator with the usual i.e., `default_rng()` approach here?

Reply:
This is done, thanks!